includes/classes/Feature/Search/Search.php

```diff
@@ -192,7 +192,16 @@ public function weight_recent( $formatted_args, $args ) {
 						'query'      => $formatted_args['query'],
 						'functions'  => array(
 							array(
-								'exp' => array(
+								/**
+								 * Filter search date weighting scale
+								 *
+								 * @hook epwr_decay_function
+								 * @param  {string} $decay_function Current decay function
+								 * @param  {array} $formatted_args Formatted Elasticsearch arguments
+								 * @param  {array} $args WP_Query arguments
+								 * @return  {string} New decay function
+								 */
+								apply_filters( 'epwr_decay_function', 'exp', $formatted_args, $args ) => array(
```

Comment:
Small suggestion - to aid readability, I would split this up like:

```
$decay_function = apply_filters( 'epwr_decay_function', 'exp', $formatted_args, $args );
...
array(
    $decay_function => array(

    )
)
```

Reply:
Sure, I dig that :+1:.